app/grandchallenge/challenges/emails.py

```diff
@@ -40,6 +40,7 @@ def send_challenge_requested_email_to_reviewers(challengerequest):
 
 def send_challenge_requested_email_to_requester(challengerequest):
     site = Site.objects.get_current()
+    link = reverse("challenges:requests-list")
```

Comment:
I can't make a comment on this line but on Line 29 you could use `User.objects.with_perm('change_challengerequest')`, bit cleaner, but check this.

Reply:
To do that we need to specify the authentication backend... That would be "django.contrib.auth.backends.ModelBackend", correct? And also, I thought it's better not to refer to User directly, but isntead use get_user_model()?

Reply:
Yes, took the example from the docs. Strange that you have to specify the backend, whatever, just leave it.